angular-express-trading-backend/models/AccountTransaction.js

```diff
@@ -0,0 +1,25 @@
+const { Model, DataTypes } = require("sequelize");
+const sequelize = require("../config/connection");
+class Transaction extends Model {}
+Transaction.init(
+  {
+    trader_id: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      primaryKey: false,
```

Comment:
technically this is a primary key since we want trader IDs to be uniquely referenced